packages/appimage/AppImageBuilder.yml

```diff
@@ -0,0 +1,106 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+version: 1
+
+script:
+  # Download, configure and make
+#  - git clone https://github.com/ygrek/mldonkey.git
+#  - cd mldonkey
+#  - ./configure --enable-batch --enable-upnp-natpmp --disable-directconnect --disable-fasttrack --disable-gnutella --disable-gnutella2 --enable-gui=newgui2 -->
+#  - make
+  # Remove any previous build
+  - rm -rf AppDir | true
+  # Get application version
+#  - ./mlnet+gui -v 2>&1 | sed -ne "s#^.*Starting MLDonkey \([^ ]\+\) .*#APP_VERSION=\1#p" >> $BUILDER_ENV
+#  - cat "$BUILDER_ENV"
+#  - APP_VERSION=$("$SOURCE_DIR"/mlnet+gui -v 2>&1 | sed -ne "s#^.*Starting MLDonkey \([^ ]\+\) .*#\1#p")
+#  - echo "$APP_VERSION"
+  # Make some needed dirs
+  - mkdir -p "$TARGET_APPDIR"/usr/bin
+  - mkdir -p "$TARGET_APPDIR"/usr/share/icons/default
+  - mkdir -p "$TARGET_APPDIR"/usr/share/applications
+  # Copy the application data into the AppDir
+  - cp "$SOURCE_DIR"/mlnet+gui                         "$TARGET_APPDIR"/usr/bin/
+  - cp "$SOURCE_DIR"/distrib/mldonkey.desktop          "$TARGET_APPDIR"/usr/share/applications
+  - cp "$SOURCE_DIR"/icons/rsvg/type_source_normal.svg "$TARGET_APPDIR"/usr/share/icons/default/mldonkey.svg
+  # Relocatable by bruteforce patching
```

Comment:
uh oh, it should be possible to build with the correct path from the start, without the need to edit the binary

Reply:
Couldn't find any current option in configure to assign the path of the executable binary.

I agree that the path should be configured without needing to resort to such ugly hacks.

Reply:
the binaries themselves can be copied, i understand the only thing that needs to be configurable is path to icons?

Reply:
No. The executable mlnet+gui has a harcoded path to /usr/bin that needed to be relocatable to run in the current directory IIRC.

For some reason the appimage didn't work placing it in /usr/bin/mlnet+gui and setting the recipe exec: to that path/file

Can recheck again when the master is fixed.

Reply:
No need to be relocatable anymore.